doc/nrf/releases/release-notes-1.4.0.rst

```diff
@@ -307,7 +307,7 @@ Thread
   * Dynamic multiprotocol support with Bluetooth LE.
   * Support for :ref:`thread_architectures_designs_cp_ncp` architecture.
   * Support for :ref:`ug_thread_vendor_hooks`.
-  * :ref:`ot_ncp_sample` sample - This sample demonstrates the usage of OpenThread Network Co-Processor architecture inside the Zephyr environment.
+  * :ref:`ot_coprocessor_sample` sample - This sample demonstrates the usage of OpenThread Network Co-Processor architecture inside the Zephyr environment.
```

Comment:
I wonder if this needs to be updated. NCS 1.4.0 had ot_ncp_sample and didn't have ot_coprocessor_sample. But if it is not changed Sphinx will probably generate an error. Maybe there is also a similar case with Known issues. 1.4 and 1.5 point to the same Known issues list but for 1.4 we should keep NCP while for 1.5 coprocessor.

@ru-fu any idea how to solve this?

Reply:
We have this behavior documented here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/release_notes.html (2nd bullet point in the note)
According to support, it should not be a big issue.